docs/tutorials/scan-a-product-tutorial.md

```diff
@@ -0,0 +1,79 @@
+# Scan A Product To Get Nutriscore
+
+This is a basic tutorial that shows you how to scan a product to get the nutriscore field. 
+
+Meet Dave. Dave is an active Open Food Facts contributor and a developer who wants to build HealthyFoodChoices, an Android app aimed at conscious consumers that buy healthy products. He has a consumer called Anna. Anna wants to know more on the nutritional facts of Nutella - Ferrero from the HealthyFoodChoices app. Dave needs his app to make an API call to provide her with this information.
+
+<!-- I am not sure If I should retain this scenario of Dave and Anna, Pierre suggested it -->
+
+## Structure of the Call
+
+To get a products nutriscore, you need to make a call to the Get A Product By Barcode Endpoint.
+
+**Authentication**
+No Authentication is required to make a query to Get A Product Nutriscore.
+
+**Describing the Request**
+Make a request to this Get A Product By Barcode endpoint.
+
+```https://world.openfoodfacts.org/api/v2/product/{barcode}
+```
+
+The {barcode} is the barcode number of the product you are trying to get. The barcode for Nutella Ferrero is 3017624010701. Then the request will be :
+
+```https://world.openfoodfacts.org/api/v2/product/3017624010701
+```
+
+This request will returning every information about Nutella Ferrero on the database. In order to get the nutri-score, we need to limit the response by specify the nutriscore field which is the `nutriscore_data`.
+<!--Is it only nutriscore_data -->
```

Comment:
Maybe you also want product_name

if we just want the nutriscore (and not the explanation of it's computation), it's enough to get nutrition_grades field.

Maybe add the nutriscore_data as a follow up to this scenario "Ana would like to be able to know how the score is computed…")

Reply:
> Maybe add the nutriscore_data as a follow up to this scenario "Ana would like to be able to know how the score is computed…")

This How-to is about the reader (i.e. the developer), so I would rephrase with "If you would like to be able to show how the score is computed...". Otherwise it means the reader needs to remember who Ana and Dave are (which one was the developer?), and that assumes the reader actually has read the introduction, otherwise it will make no sense at all.